pods-beaver-themer.php

```diff
@@ -173,29 +176,18 @@ function pods_beaver_loop_settings_before_form( $settings ) {
 	}
 
 	$setting_fields = array(
-		'use_pods'                      => array(
+		'pods_source_type'                      => array(
```

Comment:
Does changing the setting field name mean the settings won't match up with previous installs? Backcompat issues here?

Reply:
that's taken care of with `pods_beaver_update_module_settings_data_source()`  the filter runs straight after any DB call to get settings, so the correct values are cached / and saved if edited 
this maybe can be improved later on once BB releases 2.2 in a few months.

Reply:
But that doesn't handle initial load or populating the setting field? Seems like it could still break things.

Reply:
it odes it's the single point where BB get's the information out of the DB got the tip from Justin (Lead Dev @BB)

Reply:
https://www.dropbox.com/s/zgw1w5sma3bfga5/Screenshot%202018-06-19%2023.07.34.png?dl=0

Reply:
Modules will have a `filter_settings` method in 2.2. but that's at least 2 months out

Reply:
Gotcha, was just confused, it looked like it was only happening when DB was updated.